### ISSUE TITLE
website?

### ISSUE THREAD

**Issue by gabrielcsapo** [broccolijs/broccoli#373]
Currently http://broccolijs.com is being served from https://github.com/folz/broccoli-site do you intend on taking over the repo or domain?

**Comment by stefanpenner** [comment]
@gabrielcsapo, @oligriffiths and myself just had a sync up re: this:

### TL;DR

* We have essentially lost access to broccolijs.com as I haven't received responses from @joliss in-regards to restoring it. So if nothing changes, we will need to grab it once it expires...
* @trek has given us `https://broccoli.build` (which is a pretty darn cool domain)
* Today I connected the domain with https://github.com/broccolijs/broccolijs.github.io via GH Pages
* @oligriffiths plans on having a basic landing page up in the next week or so.

^^ Will give us a good place  to start. If you (or anyone else) has time/energy to lend a hand, please comment here. Specifically the broad strokes roadmap is:

### RoadMap

1. basic (and presentable) landing page with links to some good external resources/talks/slides
2. incremental improvements re: learning material on the site itself

### Ideas:

* utilize http://codesandbox.io to interactive demos/examples/learning? @michealnorth is that a good technology to build on?

**Comment by mike-north** [comment]
@stefanpenner and co

>  @mike-north you have experience with such services, would this one be appropriate for broccoli demo's/tutorials/examples?

Codesandbox would be appropriate. Here's a starter app I use for teaching

https://codesandbox.io/s/github/mike-north/broccoli-csb-starter

based on this repo https://github.com/mike-north/broccoli-csb-starter

Glitch would also work - Here's the same starter app https://glitch.com/edit/#!/silver-swift

The main benefit of CSB for this purpose is you can point it to branches, and maintain your code samples on github. Then, anyone can run a sandbox via something like
```
https://codesandbox.io/s/github/<owner>/<repo>/tree/<branch>
```

related: https://github.com/oligriffiths/broccolijs-tutorial/issues/7

**Comment by gabrielcsapo** [comment]
Awesome, I guess it would make sense to move this issue to https://github.com/broccolijs/broccolijs.github.io then right?
